syntax = "proto3";

package catalog.category;

message CategoryUpdateMessage {
  string name = 1;
  string code = 2;
  uint32 parent = 3;
  uint32 sellerId = 4;
  bool isActive = 5;
  string taxInCode = 6;
  string taxOutCode = 7;
  string incomeAccount = 8;
  string internalIncomeAccount = 9;
  string tracking = 10;
  string stockValuationAccount = 11;
  string saleBranch = 12;
  string expenseAccount = 13;
  string returnAccount = 14;
  string path = 15;
}
